dockstore-webservice/src/main/resources/import.sql

```diff
@@ -38,3 +38,6 @@ ALTER TABLE workflowversion ADD CONSTRAINT fk_workflowVersionMetadata FOREIGN KE
 
 -- cannot seem to define these due to inheritance
 ALTER TABLE tag ADD CONSTRAINT parentid_constraint FOREIGN KEY(parentid) REFERENCES tool (id) DEFERRABLE INITIALLY DEFERRED;
+
+CREATE UNIQUE INDEX unique_collection_entry ON collection_entry_version USING btree (collection_id, entry_id) WHERE version_id IS NULL;
```

Comment:
What is this file for again? This SQL is already in the migrations XML..

Reply:
I just discovered this file today.  So what happens is if I just do the migrations in the usual place, liquibase will complain and try to drop the constraint. So I had to put it here to in order to override it. I'm guessing it's what the file is for

Reply:
there's a comment above 
```
-- this file is a last chance to modify any database schema in hibernate (hbm2ddl) create mode with postgres-specific
-- changes not possible in JPA
```
basically, if you can't create something (like partial indexes) in JPA because they're postgres specific, create them here and in migrations